core/src/main/java/com/klaytn/caver/tx/kct/KIP7Token.java

```diff
@@ -0,0 +1,776 @@
+
+    public static final String FUNC_ADDMINTER = "addMinter";
+
+    public static final String FUNC_ADDPAUSER = "addPauser";
+
+    public static final String FUNC_ALLOWANCE = "allowance";
+
+    public static final String FUNC_APPROVE = "approve";
+
+    public static final String FUNC_BALANCEOF = "balanceOf";
+
+    public static final String FUNC_BURN = "burn";
+
+    public static final String FUNC_BURNFROM = "burnFrom";
+
+    public static final String FUNC_DECIMALS = "decimals";
+
+    public static final String FUNC_ISMINTER = "isMinter";
+
+    public static final String FUNC_ISPAUSER = "isPauser";
+
+    public static final String FUNC_MINT = "mint";
+
+    public static final String FUNC_NAME = "name";
+
+    public static final String FUNC_PAUSE = "pause";
+
+    public static final String FUNC_PAUSED = "paused";
+
+    public static final String FUNC_RENOUNCEMINTER = "renounceMinter";
+
+    public static final String FUNC_RENOUNCEPAUSER = "renouncePauser";
+
+    public static final String FUNC_SAFETRANSFER = "safeTransfer";
+
+    public static final String FUNC_SAFETRANSFERFROM = "safeTransferFrom";
+
+    public static final String FUNC_SUPPORTSINTERFACE = "supportsInterface";
+
+    public static final String FUNC_SYMBOL = "symbol";
+
+    public static final String FUNC_TOTALSUPPLY = "totalSupply";
+
+    public static final String FUNC_TRANSFER = "transfer";
+
+    public static final String FUNC_TRANSFERFROM = "transferFrom";
+
+    public static final String FUNC_UNPAUSE = "unpause";
+
+    public static final Event APPROVAL_EVENT = new Event("Approval",
```

Comment:
This also... EVENT_APPROVAL is more consistent compared with FUNC_XXX. Can you explain why..?

Reply:
As described above comment, I'll review all field name in this code and edit it as you suggested.